### DISCUSSION TITLE
Relationship field on column_searchable_list

### DISCUSSION THREAD

**Discussion by BhuwanPandey** [aminalaee/sqladmin#648] [Q&A]
My model looks like

from fastapi_users.db import (
    SQLAlchemyBaseUserTableUUID
)

class User(SQLAlchemyBaseUserTableUUID, Base):
    __tablename__ = "user"
    username = sa.Column(sa.String(20), unique=True, index=True)
    content = relationship("Content", back_populates="creator")

class Content(LeafBase):
    __tablename__ = "leaf_content"

    # about content
    content_title = sa.Column(sa.String(), nullable=False, server_default="UNTITLED")
    
    
    in admin.py
    
    class UserAdmin(ModelView, model=User):
          column_searchable_list = [
        User.id,
        User.content
      ]
       
by this, search doesnot work by putting content title of content . what is the usecase of accepting above scenario if it didnot search?

I think this would be a bit clear if it accept like, sothat admin panel know which field looking for

    class UserAdmin(ModelView, model=User):
          column_searchable_list = [
        User.id,
        User.content.content_title  # but this raise error 
      ]
       

Please make me clear about ths

**Comment by aminalaee** [comment]
This is not supported yet. Tracked in https://github.com/aminalaee/sqladmin/issues/643
